src/alchemlyb/parsing/amber.py

```diff
@@ -282,6 +280,12 @@ def extract_u_nk(outfile, T):
     file_datum = file_validation(outfile)
     if not file_validation(outfile):   # pragma: no cover
         return None
+
+    if not np.isclose(T, file_datum.T):
+        logger.warning(
```

Comment:
I would probably raise a Valueerror here, maybe cap the `atol` in `np.isclose` to 0.01.

Reply:
For now I added the ```atol=0.01``` parameter to ```isclose```, see the comment for WARNING/ValueError